src/vscode-bicep/schemas/bicepconfig.schema.json

```
@@ -601,6 +601,9 @@
         },
         "prettyPrinting": {
           "type": "boolean"
+        },
+        "TestFramework": {
```

[nit] lower-case the "T"

```suggestion
        "testFramework": {
```